fundamentals/arrays/lesson_notes/lesson_01.md

```diff
@@ -0,0 +1,137 @@
+
+  ```js
+  let teaOptions = ["Earl Grey", "Jasmine", "Mint"];
+  ```
+
+  > Your answer…
+
+- How can you determine the length of an array without manually counting all of the elements inside of the array?
+
+  > Your answer…
+
+- At the bottom of your Repl.it, copy and paste the following code. Before pressing the “Run” button, consider what you think will happen. Then, press “Run” to confirm your hypothesis. Did you guess right?
+
+  ```js
+  console.log([hobby1, age1, hobby2]);
+  ```
+
+- In your Repl.it, create a new variable with a name of `myArray`. Then, assign it the value of an array that includes five elements. You may use the existing variables to populate the array.
+
+  If you wanted to access the first item in the array, how would you do so?
+
+  > Your answer…
+
+- Arrays are indexed starting at 0. If your array has a length of 5, what is the index of the last element in the array? How can you use the array’s length to calculate this?
+
+  > Your answer…
+
+- Some array methods can pull items out of an array from different positions. For example, copy the code below into your Repl.it. Before running it, mentally evaluate the code and make a guess as to what will be logged. Then, run the code.
+
+  Did you guess right? What does the `.pop()` method do? What value does the `.pop()` method return?
+
+  ```js
+  let teaOptions = ["Earl Grey", "Jasmine", "Mint"];
+  let result = teaOptions.pop();
+
+  console.log(teaOptions);
+  console.log(result);
+  ```
+
+  > Your answer…
+
+- Overwrite the code above with the code below. Before running it, mentally evaluate the code and make a guess as to what will be logged. Then, run the code.
+
+  Did you guess right? What does the `.push()` method do? What values does the `.push()` method return?
+
+  ```js
+  let teaOptions = ["Earl Grey", "Jasmine", "Mint"];
+  let result = teaOptions.push("Genmai Cha", "Darjeeling");
+
+  console.log(teaOptions);
+  console.log(result);
+  ```
+
+> Your answer…
+
+- What will the following code do? Why so?
+
+  ```js
+  let teaOptions = ["Earl Grey", "Jasmine", "Mint"];
+  teaOptions.push("Chai").pop();
+  console.log(teaOptions);
+  ```
+
+  > Your answer…
+
+- It’s also possible to manually assign a spot in an array using bracket notation. Try running the following code to see how it works.
+
+  ```js
+  let teaOptions = ["Earl Grey", "Jasmine", "Mint"];
+  teaOptions[3] = "Darjeeling";
+  console.log(teaOptions);
+  ```
+
+  What would you expect to happen if instead assigning the value `"Darjeeling"` at the 3rd index, you instead did so at the 2nd? What about at 5th?
+
+  Try each of these out and then describe your findings.
+
+  > Your answer…
+
+## Exercise
+
+Clear out your Repl.it and then copy the code below into it. Underneath each comment, write a line of code that satisfies the directions.
+
+Once you are done, check-in with an instructor.
+
+```js
+let names = [];
```

Comment:
All of these are super! I like how you've baked in an ice-breaker right into the lesson